types/category.go

```diff
@@ -8,42 +8,23 @@ type Category struct {
 type CategoryName string
 
 const (
-	CategoryAppContainer          CategoryName = "app/container"
-	CategoryAppServerless                      = "app/serverless"
-	CategoryAppStaticSite                      = "app/static-site"
-	CategoryAppServer                          = "app/server"
-	CategoryCapabilityPublicEntry              = "capability/public-entry"
-	CategoryCapabilityDatastores               = "capability/datastores"
-	CategoryCapabilitySecrets                  = "capability/secrets"
-	CategoryCapabilitySidecars                 = "capability/sidecars"
-	CategoryCapabilityEvents                   = "capability/events"
-	CategoryCapabilityTelemetry                = "capability/telemetry"
-	CategoryDatastore                          = "datastore"
-	CategorySubdomain                          = "subdomain"
-	CategoryDomain                             = "domain"
-	CategoryBlock                              = "block"
+	CategoryApp        CategoryName = "app"
+	CategoryCapability              = "capability"
+	CategoryDatastore               = "datastore"
+	CategorySubdomain               = "subdomain"
+	CategoryDomain                  = "domain"
+	CategoryCluster                 = "cluster"
+	CategoryNetwork                 = "network"
+	CategoryBlock                   = "block"
 )
 
 var AllCategoryNames = []string{
-	"app/container",
-	"app/serverless",
-	"app/static-site",
-	"app/server",
-	"capability/public-entry",
-	"capability/datastores",
-	"capability/secrets",
-	"capability/sidecars",
-	"capability/events",
-	"capability/telemetry",
+	"app",
+	"capability",
 	"datastore",
 	"subdomain",
 	"domain",
+	"cluster",
```

Comment:
Remind me, why do we need cluster and network as categories?

Reply:
I added contracts to the docs site for cluster and network since they are necessary to apps and datastores.
See https://docs.nullstone.io/extending/contracts/cluster/aws/ecs.html and https://docs.nullstone.io/extending/contracts/network/aws/vpc.html

Reply:
These docs are referenced as connections from app contracts.